x/tax2gas/utils/fee_tax.go

```diff
@@ -77,7 +80,7 @@ func FilterMsgAndComputeTax(ctx sdk.Context, tk TreasuryKeeper, msgs ...sdk.Msg)
 }
 
 // computes the stability tax according to tax-rate and tax-cap
-func computeTax(ctx sdk.Context, tk TreasuryKeeper, principal sdk.Coins) sdk.Coins {
+func computeTax(ctx sdk.Context, tk types.TreasuryKeeper, principal sdk.Coins) sdk.Coins {
 	taxRate := tk.GetTaxRate(ctx)
```

Comment:
This is just a declaration, if we don't use market params's TaxRate here, no where else the TaxRate will be used cause the market module had been disabled.

Reply:
You might be correct, but what if the contract does not query tax rate? Instead, it can save a constant as taxrate inside the contract. We can't control that and those contracts need to migrate no matter what.

Reply:
So you want us to create a new BurnTaxRate params, then set the params in market module to zero right?

Reply:
Thank for your comment. We will fix that.